.github/workflows/ci.yml

```diff
@@ -66,6 +66,24 @@ jobs:
           verbose: true # optional (default = false)
       - name: prepare releasable packages
         run: make prepare
+  test-e2e:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'npm'
+      - name: install npm
+        run: npm i -g npm
+      - name: install
+        run: make install
+      - name: build
+        run: make build
+      - name: test-inspector E2E
```

Comment:
what if instead of installing, building all the app from skratch, and creating a http-server we use the package/url that we stored at s3 ?

Reply:
https://chore-e2e-tests.js-sdk-toolchain.pages.dev/inspector

Reply:
yup, I pushed some changes to be able to configure the URL via a process variable so we can use that instead, good catch!